x-pack/test_serverless/api_integration/test_suites/common/security/api_keys.ts

```diff
@@ -0,0 +1,206 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from 'expect';
+import { FtrProviderContext } from '../../../ftr_provider_context';
+
+export default function ({ getService }: FtrProviderContext) {
+  const svlCommonApi = getService('svlCommonApi');
+  const supertest = getService('supertest');
+  let roleMapping: { id: string; name: string; api_key: string; encoded: string };
+
+  describe('security/api_keys', function () {
+    describe('route access', () => {
+      describe('internal', () => {
+        // The create test MUST be executed for the update, get, get all, and invalidate tests to function
+        it('create', async () => {
+          let body: any;
```

Comment:
__nit:__ can we use `unknown` here and in the test below?
```suggestion
          let body: unknown;
```